src/AddressRegistry/StreetName/StreetName.cs

```diff
@@ -162,6 +171,152 @@ public void CorrectStreetNameRetirement()
             }
         }
 
+        [EventTags(EventTag.For.Edit, EventTag.For.Sync)]
+        [EventName(EventName)]
+        [EventDescription("Het adres werd geheradresseerd.")]
+        public class StreetNameWasReaddressed : IStreetNameEvent
+        {
+            public const string EventName = "StreetNameWasReaddressed"; // BE CAREFUL CHANGING THIS!!
+
+            [EventPropertyDescription("Objectidentificator van de straatnaam aan dewelke het adres is toegewezen.")]
+            public int StreetNamePersistentLocalId { get; }
+
+            [EventPropertyDescription("De voorgestelde adressen.")]
+            public IReadOnlyList<int> ProposedAddressPersistentLocalIds { get; }
+            [EventPropertyDescription("De heradresseerde adressen.")]
+            public IReadOnlyList<ReaddressAddressData> ReaddressAddresses { get; }
```

Comment:
Readdress**ed**Addresses instead of ReaddressAddressData? The latter sounds like a command.